src/main/resources/static/password-reset/reset.html

```
@@ -0,0 +1,48 @@
+                <input type="password" id="newPassword" placeholder="Enter a new password" autocomplete="new-password" />
+            </div>
+
+            <button id="applyResetBtn" class="reset-button">Reset Password</button>
+
+            <div id="level7ExternalCard" class="external-card hidden">
+                <h5>External Resource</h5>
+                <p class="external-copy">Level 7 intentionally loads a third-party image so the browser may send the full reset-link URL as the HTTP referrer.</p>
+                <img
+                    id="level7ExternalImage"
+                    class="external-image"
+                    src=""
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)